test/e2e/framework/helper/certificates.go

```diff
@@ -71,15 +71,9 @@ func (h *Helper) waitForCertificateCondition(client clientset.CertificateInterfa
 	if pollErr != nil && certificate != nil {
 		log.Logf("Failed waiting for certificate %v: %v\n", name, pollErr.Error())
 
-		if len(certificate.Status.Conditions) > 0 {
-			log.Logf("Observed certificate conditions:\n")
-			for _, cond := range certificate.Status.Conditions {
-				log.Logf("- Last Status: '%s' Reason: '%s', Message: '%s'\n", cond.Status, cond.Reason, cond.Message)
-			}
-		}
+		log.Logf("Certificate:\n")
+		h.describeCMObject(certificate)
```

Comment:
Does this make it easier to read? Previously this would have run `kubectl describe certificate` right? In which case it would have included events related to the cert. Will we miss that?

Would the old `kubectl describe` output have gone to stderr?

Reply:
I do it this way, so we are certain the certificate displayed is the certificate that we were working with. Otherwise, it might be an updated version of the certificate.

Reply:
instead of re-fetching the cert; I just display the current certificate.